commands/Moderation/ban.js

```diff
@@ -41,11 +41,12 @@ class Ban extends Command {
             reason = message.language.get("UTILS").NO_REASON_PROVIDED;
         }
         
-        await user.send(message.language.get("BAN_SUCCESS_DM", user, message, reason)).catch((err) => {});
```

Comment:
Il faut envoyer le message avant, sinon le bot ne peut plus (Cannot send message to this user) sauf si l'utilisateur se trouve sur un autre de ses serveurs, ce qui arrive rarement

Reply:
je n'avais pas pensais a ca..